package-lock.json

```diff
@@ -127,9 +127,9 @@
       }
     },
     "node_modules/@babel/generator": {
-      "version": "7.20.2",
-      "resolved": "https://registry.npmjs.org/@babel/generator/-/generator-7.20.2.tgz",
-      "integrity": "sha512-SD75PMIK6i9H8G/tfGvB4KKl4Nw6Ssos9nGgYwxbgyTP0iX/Z55DveoH86rmUB/YHTQQ+ZC0F7xxaY8l2OF44Q==",
+      "version": "7.20.4",
+      "resolved": "https://registry.npmjs.org/@babel/generator/-/generator-7.20.4.tgz",
+      "integrity": "sha512-luCf7yk/cm7yab6CAW1aiFnmEfBJplb/JojV56MYEK7ziWfGmFlTfmL9Ehwfy4gFhbjBfWO1wj7/TuSbVNEEtA==",
```

Comment:
Is it normal that the package-lock.json has changed on this PR without an update on `package.json`?

Reply:
no it's not I think the rebase messed everything up 😢 
I'll try to fix it